### DISCUSSION TITLE
Automatic signals selection from signal group header

### DISCUSSION THREAD

**Discussion by MaceWindu** [datalust/seq-tickets#2283] [Ideas]
I have a signal group to allow me to see events from all signals in group merged, but now I need to select each signal manually.

Would be nice to make it better by adding an option to (de)select all signals in group in single click. E.g. using additional button/tri-state ckeckbox near to group name?

**Comment by nblumhardt** [comment]
Thanks for the suggestion; we're expecting to loop back around to look at signal bar design in a release or two so we'll keep this in mind.
